modules/API/API.js

```diff
@@ -87,6 +87,17 @@ function initCommands() {
 
             APP.store.dispatch(muteAllParticipants(localIds));
         },
+        'mute-participant-audio': participantId => {
+            sendAnalytics(createApiEvent('muted-participant-audio'));
+            const state = APP.store.getState();
+            const localId = getLocalParticipant(state).id;
+
+            if (participantId === localId) {
+                muteLocal(true);
```

Comment:
`muteLocal` and `muteRemote` are actions. In order to make them work you'll need to `dispatch` them!

Also if you check they send analytics events that indicate button presses on the remote menu. It will be nice to create new actions that will send more appropriate analytics events. Maybe you can put the new actions in react/features/external-api.